src/azure-cli/azure/cli/command_modules/synapse/tests/latest/test_synapse_scenario.py

```diff
@@ -2985,3 +2985,73 @@ def test_sqlscript(self):
             'az synapse sql-script show --workspace-name {workspace} --name {name}',
             expect_failure=True)
 
+    @record_only()
+    def test_link_connection(self):
+        self.kwargs.update({
+            'workspace_name': 'xiaoyuxingtestne',
+            'link_connection_name': 'linkconnectionfortest',
+            'link_table_id': '887e9d4df0fa4afaaad0d7a2c7f42d88',
```

Comment:
Using hard coded resources is not recommended. Others may not able to rerun this test due to access failure of these resources.

Is it possible to refine the test and remove `record_only()`?

Reply:
Because this link connection required two sql database and each database needs to create a table. Besides "link_table_id" is a random string. This workspace and its database is hard coded resource.